draft-ietf-lake-edhoc.md

```diff
@@ -1122,7 +1153,9 @@ An implementation MAY support only Initiator or only Responder.
 
 An implementation MAY support only a single method. None of the methods are mandatory-to-implement.
 
-Implementations MUST support 'kid' parameters of type int. None of the other COSE header parameters are mandatory-to-implement.
+Implementations MUST support byte string identifiers represented as one-byte CBOR ints, as described in {{bstr-repr}}.
```

Comment:
> Implementations MUST support byte string identifiers represented as one-byte CBOR ints, as described in {{bstr-repr}}.

If we have MUST earlier in the document I think this sentence should be removed. "supported" gives the idea that this is not mandatory to encode as int, which would lead to a non-deterministic encoding.